JS/HW-17/index.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+// let first = 'true';
+// let second = false;
+// let third = 17;
+// let fourth = undefined;
+// let fifth = null;
+// console.log(typeof(first), typeof(second), typeof(third), typeof(fourth), typeof(fifth));
+
+// let height = 15;
+// let width = 20;
+// if (width > height) {
+//     console.log(width);
+// } else {
+//     console.log(height);
+// }
+
+// let i = 1;
+// while(i <= 20) {
+//     if(i % 3 == 0){
+//         console.log(i);
+//     }
+//     i+=1;
+// }
+
+// let key = true;
+// let documents = true;
+// let pen = true;
+// let apple = false;
+// let orange = true;
+// let shoulGoToWork;
+// if((key == true && documents == true && pen == true) && (apple == true || orange == true)){
```

Comment:
Строгое равенство. Первые скобки не обязательны. Пробел перед {
```suggestion
// if (key === true && documents === true && pen === true && (apple === true || orange === true)) {
```